### DISCUSSION TITLE
記事タイトルに、その記事自身へのリンク を追加してほしい

### DISCUSSION THREAD

**Discussion by KSR-Yasuda** [increments/qiita-discussions#894] [要望・アイデア・不具合]
記事のタイトルに、その記事自身へのリンク を追加できないでしょうか？

Chromeで、リンクのコンテキストメニュー経由で、記事ページを Chromeのリーディングリスト に追加するためです。

Android環境で、メーリングリストから、メーラーのアプリ内ブラウザ で記事を開いたとき、
その 開いているページ自身 を リーディングリスト に追加する方法がありません。
ただし、リンクをロングタップ → コンテキストメニューから リーディングリストに追加 はできるので、
その記事自身へのリンク があると、助かるのです。

一旦 Chromeで 記事ページを開き直せば、開いているページ自身をリーディングリスト追加できるのですが、
可能なら、アプリ内ブラウザ で完結したいです。

**Comment by acronhub** [comment]
@KSR-Yasuda
リクエストありがとうございます。

お使いのメーラーはGmailでしょうか？
その場合、調査したところGmail アプリ内のブラウザであれば、Chrome のリーディングリストへの追加を行うことができましたので、下記手順を参考にしていただけると幸いです。

手順
1. Gmail アプリを開きます
2. メールの本文にあるリンクをタップして、アプリ内ブラウザでページを開きます
3. ページを開いた状態で、右上のその他アイコン︙ > スターアイコン ☆ をタップします
4. ブックマークの編集画面で、[フォルダ] > [リーディング リスト] を選択します

Chromeの場合の「リーディング リストにページを保存する」が同じような操作方法でしたので、詳細は下記をご参照ください。
https://support.google.com/chrome/answer/7343019?hl=ja&co=GENIE.Platform%3DAndroid

**Reply by KSR-Yasuda** [reply]
@acronhub あ、なるほど、ここにあるんですね。
︙マークのメニューが少ないのに気を取られて、☆マークに気付いてませんでした。

本件、ご教示の方法で リスト追加ができましたので、closeします。
ありがとうございました。
